### PR TITLE
Fix formatting in install doc page for MetalLB

### PR DESCRIPTION
**Pull request by sandeepkanabar**
Fix for https://github.com/metallb/metallb/issues/2222. This PR fixes the formatting issue due to `{{% notice note %}}` wherein `Using The MetalLB Operator` section note was **overflowing** into `FRR Daemons Logging Level` section.

Thanks for sending a pull request! A few things before we get started:

1. If this is your first time, please read the [contributing guide](https://metallb.universe.tf/community/#contributing)
2. For non-trivial pull requests, please [file an
   issue](https://github.com/metallb/metallb/issues/new) first, and get
   agreement that the change is a good idea, and a general guideline
   for how it should be implemented, before sending code. Large PRs
   that weren't first discussed and agreed upon in an issue won't be
   accepted.
3. If the PR fixes a particular bug, please include the words "Fixed
   #<issue number>" in the PR text, so that the bug auto-closes when
   the PR is merged.

### CODE REVIEW

**Comment by fedepaol** [comment]
@sandeepkanabar mind changing the commit message with what you did and why it solves the issue?
A good reference of a commit subj / body is http://chris.beams.io/posts/git-commit

File: website/content/installation/_index.md [hunk 1]
```diff
@@ -94,7 +94,6 @@ service account. It is suggested to use either the charts or kustomize if they
 need to be changed.
 
 {{% notice note %}}
-
```
Comment (fedepaol) [inline website/content/installation/_index.md:97]: is this related to the issue?

**Comment by sandeepkanabar** [comment]
> @sandeepkanabar mind changing the commit message with what you did and why it solves the issue? A good reference of a commit subj / body is http://chris.beams.io/posts/git-commit

Thanks a ton for the excellent link. Much appreciated @fedepaol.

File: website/content/installation/_index.md (hunk 1 above)
Comment (sandeepkanabar) [reply website/content/installation/_index.md:97]: @fedepaol  - no this is not related to the issue but I'm not 100% sure. This is more of changing it to match the other `notice note` tags. If you look at the other `{{% notice note %}}` tags, you'll see that the text following it immediately begins in the new line while in this case there was an extra new line. Feel free to correct me if I'm wrong here.

**Comment by sandeepkanabar** [comment]
> @sandeepkanabar mind changing the commit message with what you did and why it solves the issue? A good reference of a commit subj / body is http://chris.beams.io/posts/git-commit

Done. Have modified the commit message to reflect the changes. Please have a look and LMK your thoughts. Feedback is much appreciated.

**Comment by sandeepkanabar** [comment]
@fedepaol - would you mind reviewing this when you get some time. Thank you.

**Comment by fedepaol** [comment]
> > @sandeepkanabar mind changing the commit message with what you did and why it solves the issue? A good reference of a commit subj / body is http://chris.beams.io/posts/git-commit
> 
> Thanks a ton for the excellent link. Much appreciated @fedepaol.

I still see a non-descriptive commit message though. Can you please elaborate on why this is fixing the issue and not refer to the issue itself? Thanks!
Also, the subject is quite generic.

File: website/content/installation/_index.md [hunk 2]
```diff
@@ -206,7 +205,9 @@ named `metallb-operator`:
 kubectl edit csv metallb-operator
 ```
 
-and change the `BGP_TYPE` environment variable of the `manager` container to `frr`:
+{{% /notice %}}
```
Comment (fedepaol) [inline website/content/installation/_index.md:208]: This would actually split the note in 2, cutting the text. Not sure it's what we want
